Оформление

Nikita7515 opened the pull request.

Nikita7515 committed:
Подключение шрифтов

Создание styles и style.css . Форматирофание длинных строк.

Nikita7515 committed:
body и ссылки страниц

Nikita7515 committed:
доработка изображений

Nikita7515 committed:
Кнопка в шапке  и hero блок

Nikita7515 committed:
Заголовок, body и блоки с преимуществами,доработка index.html

Nikita7515 committed:
доработки блока преимуществ

Nikita7515 committed:
Блок подписки на рассылку

и дооформил body

Nikita7515 committed:
Добавление картинок и доработка мелочей

Nikita7515 committed:
Доработка размеров у изображений

Nikita7515 committed:
правки

Nikita7515 committed:
Стилизация hero в catalog.html

И дополнительные правки

Nikita7515 committed:
Стилизация списка просмотра и карточек

Nikita7515 committed:
Оформление пагинации

Nikita7515 committed:
правки background-size

Nikita7515 committed:
Переделка visibility-hidden

Nikita7515 committed:
Убрал иконки скрытых элементов

summertime-fms commented on styles/styles.css:
```diff
@@ -0,0 +1,288 @@
+}
+
+@font-face {
+  font-family: "PT Sans", Arial, sans-serif;
```
ариал тоже в кавычках давай

summertime-fms commented on styles/styles.css:
```diff
@@ -0,0 +1,288 @@
+  background-color: rgba(242, 242, 242, 1);
+}
+
+body {
+  margin-left: auto;
+  margin-right: auto;
+  background-color: rgb(255, 255, 255);
+  width: 1200px;
```
давай ка лучше бади не будем трогать. для контентной части создадим div с классом wrapper и вот он уже будет 1200 шириной и отцентрован. и в нем будем лежать вся верстка которая сейчас в бади.

Nikita7515 committed:
Доработки

шрифты доработал и вместо черного цвета остальным элементам, просто сделал им цвет наследуемым.

Nikita7515 committed:
Merge branch 'second'

Nikita7515 committed:
Доработка стилей

подумал над переполнением

Nikita7515 committed:
Сделан wrapper в body и отцентрован
